app/views/meals/setup.html.erb

```diff
@@ -7,4 +7,12 @@
     <% count += 1 %>
     <%= image_tag "https://source.unsplash.com/414x736/?#{keyword}" %>
   </div>
+  <div>
+    <%= simple_form_for(@poll) do |f| %>
+      <%= f.input :score  %>
+      <%= f.hidden_field :cuisine, value:  cuisine %>
```

Comment:
This line will pass cuisine as a string, not cuisine instance. So I passed cuisine id instead